templates/shortcodes/donate_buttons.html

```diff
@@ -3,26 +3,26 @@
   <a href="https://donate.organicmaps.app?currency=rub" title="По карте МИР в рублях"><img id="mir" src="/images/donates/mir.svg" alt="По карте МИР в рублях"></a>
   <a href="https://t.me/OrganicMapsRu/29441" title="Telegram по карте МИР в рублях"><img id="telegramRu" src="/images/donates/telegram.svg" alt="Telegram по карте МИР в рублях"></a>
   {% endif -%}
-  <a href="{{ config.extra.stripe | safe }}" title="Credit/Debit Card"><img src="/images/donates/visa_mc.svg" alt="Credit/Debit Card"></a>
-  <a href="{{ config.extra.stripe | safe }}" title="Apple Pay" id="applePay"><img src="/images/donates/apple.svg" alt="Apple Pay"></a>
-  <a href="{{ config.extra.stripe | safe }}" title="Google Pay" id="googlePay"><img src="/images/donates/google.svg" alt="Google Pay"></a>
+  <a href="https://donate.organicmaps.app/?locale={{ lang }}" title="Credit/Debit Card"><img src="/images/donates/visa_mc.svg" alt="Credit/Debit Card"></a>
```

Comment:
1. Do all our langs match stripe langs?
2. What is the stripe behavior when a lang is not supported there?
3. Is it possible to enable automatic lang detection without passing an explicit parameter, as it was mentioned on the stripe's documentation page?

Reply:
The problem is that this approach requires some code/support, and may introduce unexpected issues with unsupported languages/wrong parameters, while automatic detection by stripe works automatically and doesn't require any changes.

What are the benefits enabled by merging this PR?

Reply:
> his PR just do all the best to continue with the same language as seen on the web-site. What is the problem with that?

The wrong locale on our website will prohibit Stripe from autodetecting the correct locale that is not supported yet on our website. For example, a Greek user will see en OM website and will see en Stripe instead of el.


P.S. A shortcode can help to avoid copy-paste.

Reply:
Any unsupported language will fall back to en on our website. I suggest to use automatic stripe language detection logic for en in this case, to help users see localized stripe pages when their languages are not yet supported on our website.